### ISSUE TITLE
Removed from Firefox Store?

### ISSUE THREAD

**Issue by Mx772** [tfedor/IsThereAnyDeal-Everywhere#15]
Saw the URL on the website/readme links to:
https://addons.mozilla.org/en-US/firefox/addon/IsThereAnyDeal-Everywhere/

Which returns:
![image](https://github.com/user-attachments/assets/6bf014da-b43b-421e-bb82-5799c8b194b1)

Not sure if intentional or what.

**Comment by tfedor** [comment]
Yes, I unlisted it. I don't have capacity to work on the extension at the moment.

**Comment by Mx772** [comment]
Gotcha - Saw it was still available on Chrome so wasn't fully sure. 

Are you open to contributors like: https://github.com/IsThereAnyDeal/Everywhere/pull/14

Or having someone else host it on firefox? Just for signing/not having to run it as a temporary plugin. 

If not, completely understand!
